packages/terra-form-select/src/native-select/NativePropTypes.js

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
+
+/**
+ * NOTE: Changes to this file should be mirrored in the `terra-dev-site/doc/example/propTypes` folder
+ */
+
+const optionPropType = PropTypes.shape({
+  /**
+   * The option display.
+   */
+  display: PropTypes.string.isRequired,
+  /**
+   * Whether the option is disabled.
+   */
+  disabled: PropTypes.bool,
+  /**
+   * The option value.
+   */
+  value: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
+});
+
+const optGroupPropType = PropTypes.shape({
+  /**
+   * The option display.
+   */
+  display: PropTypes.string.isRequired,
+  /**
+   * Whether the optgroup is disabled.
+   */
+  disabled: PropTypes.bool,
+  /**
+   * The array of select options.
+   */
+  childOptions: PropTypes.arrayOf(optionPropType).isRequired,
```

Comment:
How do you feel about naming this `options`? Did you chose this to avoid the clash with the root `options`?

Reply:
I did change it so it was a little more explicit, but `options` makes sense too. Do you prefer `options`?

Reply:
Updated.